src/index.tsx

```diff
@@ -137,15 +139,25 @@ export default class InfiniteScroll extends Component<Props, State> {
     }
   }
 
-  UNSAFE_componentWillReceiveProps(props: Props) {
+  componentDidUpdate(nextProps: Props) {
```

Comment:
@Newbie012 - Don't you think `prevProps` would be a better name here following the official documentation?
https://reactjs.org/docs/react-component.html#componentdidupdate

Reply:
You're absolutely right. I have changed it to prevProps.